cypress/integration/SingleSelect/common/index.js

```diff
@@ -0,0 +1,27 @@
+import { Given, Then } from 'cypress-cucumber-preprocessor/steps'
+
+Given('a SingleSelect with options is rendered', () => {
+    cy.visitStory('SingleSelect', 'With options')
+})
+
+Given('a SingleSelect with options and a selection is rendered', () => {
+    cy.visitStory('SingleSelect', 'With options and a selection')
+})
+
+Given('an onChange handler is attached', () => {
+    cy.window().then(win => (win.onChange = cy.stub()))
+})
+
+Given('the SingleSelect is open', () => {
+    cy.get('.select [tabIndex="0"]').click()
+
+    cy.contains('option one')
+    cy.contains('option two')
+    cy.contains('option three')
```

Comment:
I'd add the `.should('exist')` here as well

Reply:
Ok, done